### PR TITLE
Add the arm64 image to the manifest create command

### PR DESCRIPTION
**Pull request by upodroid**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fixing something I missed in #1369 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by upodroid** [comment]
Can we cherrypick this to 1.11 release branch and cut a release please?

File: CHANGELOG/CHANGELOG-1.11.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-# v1.11.0 - Changelog since v1.10.5
+# v1.11.2 - Changelog since v1.11.1
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-1.11.md:1]: This gets generated when a new release is cut. So this file should be reverted.

Comment (upodroid) [reply CHANGELOG/CHANGELOG-1.11.md]: Done

**Comment by upodroid** [comment]
The change worked.

```
 mahamed  REDACTED  ~  Desktop  Git  $  gcrane manifest gcr.io/k8s-staging-cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver:master
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 8734,
         "digest": "sha256:88295e6e807896f7938b5951e7796e22857fc30f6bd77b8eb5649ef6ad241936",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 8734,
         "digest": "sha256:377de31e12c2a7a9b5db0f92d2880a5e842aa4dceb41600dc2af2809fd4a9ae1",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 905,
         "digest": "sha256:29950a2f650d67cebca62730eedc6824f36ab64c4e1394c7967db156b1900ffa",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.17763.4851"
         }
      }
   ]
}
```

**Pull request merged**